distribution/Containerfile

```diff
@@ -6,46 +6,46 @@ WORKDIR /opt/app-root
 
```

Comment:
There shouldn't be this many changes to the Containerfile

Reply:
using the `set` shuffles the whole req.txt :/

Reply:
Resolving this thread since you've started a new one below